build.gradle

```diff
@@ -30,7 +30,7 @@ allprojects {
         getCurrentShortRevision = {
             new ByteArrayOutputStream().withStream { os ->
                 exec {
-                    executable = "git"
+                    executable = "git.bat"
```

Comment:
Is this some kind of local change you made? We don't build on Windows.

Reply:
Didn't mean to push this, my bad. Signs of my janky WSL.

Reply:
This is still live. Can you still revert this?